doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -203,17 +203,35 @@ The [Cinder Configuration]( https://docs.openstack.org/cinder/latest/configurati
 
 [Ceph](https://ceph.io/) is the default CNTT Reference Architecture storage backend and is discussed below.
 
-#### 4.2.4.1. Ceph
+#### 4.2.4.1. Ceph Storage Cluster
+The Ceph storage cluster is deployed on bare metal hardware. The minimal configuration is a cluster of three bare metal servers to ensure High availability. The Ceph Storage cluster consists of 3 main daemon types:
+-	Monitor
+-	OSD (object storage daemon)
+-	RadosGW (Rados Gateway)
 
-**Content to be developed**
--	Types of storage (incl NVMe)
--	Ceph
-    -	Ceph Configuration
-    -	Cinder Gateway, Swift Gateway
--	How many nodes to meet SLA
--	Sizing rules (IOPS, SSD vs. SATA, NICs, etc).
+Ceph monitors maintain a master copy of the maps of the cluster state required by Ceph daemons to coordinate with each other. Ceph OSD handle the data storage (read/write data on the physical disks), data replication, recovery, rebalancing, and provides some monitoring information to Ceph Monitors. The RadosGW provides Object Storage RESTful gateway with a Swift-compatible API for Object Storage.
+
+<p align="center"><img src="../figures/Figure_4_x_Ceph.png" alt="Ceph Storage System"></br>Figure 4-2. Ceph Storage System.</p>
+
+**BIOS Requirement for Ceph servers**
 
+| BIOS/boot Parameter | Control Srever |
+|-------------|----------------|
+| Boot disks | RAID 1 |
 
+How many nodes to meet SLA :
+-	minimum: three bare metal servers where Monitors are collocated with OSD. Note: at least 3 Monitors and 3 OSDs are reuired for High AVailability. 
+
+HW specifications :
+-	the boot disks are SSD disks with a minimum capacity of 120GB
+-	for an IOPS oriented cluster (full SSD),  the journal can be hosted on OSD disks
+-	for a capacity oriented cluster (HDD), the journal must be hosted on dedicated SSD disks
```

Comment:
based on IOPS required, so if we need to have high IOPS we need to have either SSD or NVMe 
and for Journal, there is a practical  recommended ratio, if we have HDD OSD so we need to have 4:1 Journal ration 
if we have SSD OSD so ration may be 10:1 


Right now there is optane SSD , Did you think we need to mention it ? @pgoyal01

Reply:
@ASawwaf agree need to mention it: @samuelhellec?

Reply:
I am afraid that we go in this level of detail it will be hard to maintain because disks technology will evolve .
so maybe  we could re sentence like this :

HW specifications :
-	the boot disks are dedicated with Flash technology disks
-	for an IOPS oriented cluster (Flash technology ),  the journal can be hosted on OSD disks
-	for a capacity oriented cluster (HDD), the journal must be hosted on dedicated Flash technology disks

@pgoyal01 @ASawwaf  I you agree with this point of view we will need to review the document with this angle and maybe write a sentence regarding the terminology of flash and HDD

Reply:
@samuelhellec Agree. Can you make the changes. Thanks

Reply:
done